judgments/migrations/0002_alter_courtdates_options.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.2.22 on 2023-11-09 05:34
```

Comment:
This is a bit concerning; where does this change come from?

Reply:
ok, i think there was an earlier change somewhere on `main` which would have generated a migration, but for whatever reason we never ran the migration generation command at the time, so therefore when we added waffle and ran the migrations for that, it generated and applied the old migration too. Maybe this should come out of this PR and I'll open a new one for it on its own?

Reply:
As discussed, i'll open another PR with that change!